### PR TITLE
Fix underscore linting issue in /pkg/volume/azure_dd/azure_common_lin…

### PR DESCRIPTION
**Pull request by hazel-shen**
…ux.go

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:

> /kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by matthyx** [comment]
@hazel-shen can you rebase your PR?
/cc

**Comment by hazel-shen** [comment]
> @hazel-shen can you rebase your PR?
> /cc

@matthyx 
I've also fixed the conflict, please take a look, thank you!

File: pkg/volume/azure_dd/azure_common_linux.go [hunk 1]
```diff
@@ -76,14 +76,14 @@ func scsiHostRescan(io ioHandler, exec utilexec.Interface) {
 	scsi_path := "/sys/class/scsi_host/"
```
Comment (matthyx) [inline pkg/volume/azure_dd/azure_common_linux.go]: Can you change it here too?

File: pkg/volume/azure_dd/azure_common_linux.go [hunk 2]
```diff
@@ -76,14 +76,14 @@ func scsiHostRescan(io ioHandler, exec utilexec.Interface) {
...
 	if dirs, err := io.ReadDir(scsi_path); err == nil {
```
Comment (matthyx) [inline pkg/volume/azure_dd/azure_common_linux.go]: And here?

**Review by matthyx — CHANGES_REQUESTED** [review]

**Pull request closed without merging**